WBooks/Screens/MainViewController.swift

```diff
@@ -0,0 +1,60 @@
+//
+//  MainViewController.swift
+//  WBooks
+//
+//  Created by Nacho 2 on 20/01/2020.
+//  Copyright © 2020 Wolox. All rights reserved.
+//
+
+import UIKit
+import WolmoCore
+
+class MainViewController: UIViewController, UITableViewDelegate, UITableViewDataSource {
+    private let _view: MainView = MainView.loadFromNib()!
+    
+    var booksArray : Array = [["image": "img_book2", "title": "The best book in the world", "subtitle": "Peter Sjermstrom"], ["image": "img_book3", "title": "A little bird told me", "subtitle": "Timpthy Cross"], ["image": "img_book4", "title": "When the doves desappeared", "subtitle": "Sofi Oksanen"]]
+
+    func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        booksArray.count
+    }
+    
+    func tableView(_ tableView: UITableView, heightForRowAt indexPath: IndexPath) -> CGFloat {
+        100
```

Comment:
Is this size enough if for example, the name of the book takes 3 lines?

Reply:
Title is set for two lines max

![image](https://user-images.githubusercontent.com/24418053/72839156-8266cb80-3c70-11ea-9af9-f8b05d131a9a.png)


subtitle just one line

Thats ok?

Reply:
You can make the cell grow based on it's content, but it's ok for this